### PR TITLE
Magento1 doc - installation steps update

### PR DESCRIPTION
**Pull request by bastienlm**

### CODE REVIEW

File: source/docs/magento1/installation.md [hunk 1]
```diff
@@ -96,15 +96,19 @@ If the installation is successful, in Magento's administration panel, you will h
...
 ### **Magento core patch**
 
-  If the OAuth Zend Patch is valid in the installer checker, please follow these steps:
+  If the OAuth Zend Patch is not valid in the installer checker, please follow these steps:
   - Copy the [`fix-sort-params-core.patch`](https://gitlab.com/front-commerce/magento1-module-front-commerce/blob/master/fix-sort-params-core.patch) file in your root directory
   - Past it on your root Magento directory
-  - Apply them `git apply fix-sort-params-core.patch`
+  - Apply them
+    - Whit GIT: `git apply fix-sort-params-core.patch`
+    - Whitout GIT: `patch -p1 < fix-sort-params-core.patch`
```
Comment (real34) [inline source/docs/magento1/installation.md]:
```suggestion
    - Whithout GIT: `patch -p1 < fix-sort-params-core.patch`
```

Comment (bastienlm) [reply source/docs/magento1/installation.md]: Without* :smile:

**Comment by bastienlm** [comment]
PR Updated :+1: thanks for the review.

**Comment by real34** [comment]
Sorry for the delay, I did not noticed your update.

**Pull request merged**